.kitchen.yml

```diff
@@ -0,0 +1,54 @@
+# vim: ft=yaml
+---
+driver:
+  name: vagrant
+  customize:
+    memory: 512
+  vm_hostname: postfix.ci.local
+
+provisioner:
+  name: salt_solo
+  # As of 2016/06 http://bootstrap.saltstack.org do not handle ubuntu 16.04. and fails in docker container
+  # salt_bootstrap_url: https://bootstrap.saltstack.org
+  salt_bootstrap_url: https://raw.githubusercontent.com/epcim/salt-bootstrap/add-apt-transport-https-ubuntu/bootstrap-salt.sh
+  salt_install: bootstrap
+  salt_version: latest
+  formula: postfix
+  log_level: info
+  state_top:
+    base:
+      "*":
+        - postfix
+  pillars:
+    top.sls:
+      base:
+        "*":
+          - postfix
+    postfix.sls:
+      server:
+        ssl:
```

Comment:
this is already default afaik (ssl.enabled=false + there's a typo enable vs enabled bellow)